packages/dapp/src/components/Navbar/NavbarMenuDesktop.tsx

```diff
@@ -63,49 +76,37 @@ const NavbarMenuDesktop = ({
             <Grow
               {...TransitionProps}
               style={{
-                transformOrigin: 'right top',
+                transformOrigin: transformOrigin ? transformOrigin : 'top',
```

Comment:
`transformOrigin || 'top'`

Reply:
Was found in 14 files! Refactored it, much better style